### PR TITLE
📖 docs(amp-date-countdown): add README for bento component

### PR DESCRIPTION
**Pull request by kvchari**
continues https://github.com/ampproject/amphtml/pull/35874

adds README for bento component

### CODE REVIEW

File: extensions/amp-date-countdown/1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,318 @@
+---
+$category@: presentation
+formats:
+  - websites
+teaser:
+  text: Displays a countdown sequence to a specified date.
+experimental: true
+bento: true
```
Comment (caroqliu) [inline extensions/amp-date-countdown/1.0/README.md:8]: this can be removed

File: extensions/amp-inline-gallery/1.0/README.md [hunk 2]
```diff
@@ -216,7 +216,3 @@ Number: ratio of width to height that slides should be displayed in.
 Default: `false`
 
 Boolean attribute indicating whether thumbnails should loop.
-
```
Comment (caroqliu) [inline extensions/amp-inline-gallery/1.0/README.md:219]: This PR and #36021 both have some extraneous edits to this file. Do you think it's worth consolidating changes to this file and the amp-stream-gallery documentation into a single PR?

**Review by caroqliu — APPROVED** [review]

File: extensions/amp-inline-gallery/1.0/README.md (hunk 2 above)
Comment (kvchari) [reply extensions/amp-inline-gallery/1.0/README.md:219]: sure

File: extensions/amp-date-countdown/1.0/README.md (hunk 1 above)
Comment (kvchari) [reply extensions/amp-date-countdown/1.0/README.md:8]: should I remove both the `experimental` and the `bento` properties? it looks like the other README should be updated as well

File: extensions/amp-inline-gallery/1.0/README.md (hunk 2 above)
Comment (kvchari) [reply extensions/amp-inline-gallery/1.0/README.md:219]: closing this PR and merging the commits from this branch into the date-display PR #36021

**Pull request closed without merging**